src/app/api/utils/email.tsx

```diff
@@ -10,14 +10,6 @@ import { VerifyEmailAddressEmail } from "../../../emails/templates/verifyEmailAd
 import { sanitizeSubscriberRow } from "../../functions/server/sanitize";
 import { getL10n } from "../../functions/l10n/serverComponents";
```

Comment:
This import causes problems in cronjobs (which is why there's a separate l10n module for cronjobs), and since this `unsubscribeLinkForSubscriber` is only intended for the monthly free email, I moved it into that module.

Reply:
Do you know why this is the case? I remember we used to have to do that `localstorage` workaround but that's been removed.

Reply:
Yes, this is the error message:

```
ReferenceError: require is not defined in ES module scope, you can use import instead
    at <anonymous> (/<snip>/blurts-server/src/app/functions/l10n/serverComponents.ts:23:30)
```

Specifically, [we use a Webpack API](https://github.com/mozilla/blurts-server/blob/ba1dfdebe5fefad6edad50a32da78fb838b9494a/src/app/functions/l10n/serverComponents.ts#L21-L23) (which will work both on the server-side as well as on the client-side) to load FTL files in Next.js, but that API is not available in cronjobs. Thus, in cronjobs, we just [use the plain Node APIs](https://github.com/mozilla/blurts-server/blob/ba1dfdebe5fefad6edad50a32da78fb838b9494a/src/app/functions/l10n/cronjobs.ts#L34).

(Differences between runtimes like these are why I used to argue for splitting up the projects, but I think we've got it working pretty well now - but you'll still see symptoms of the way we need to work around the differences here and there, and this is one example.)